src/_includes/graphql/return.md

```diff
@@ -0,0 +1,65 @@
+The `Return` object can contain the following attributes:
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`available_shipping_carriers` | [ReturnShippingCarrier] | A list of shipping carriers available for returns
+`comments`| [ReturnComment] | A list of comments posted for the return request
+`creation_date` | String! | The date the return was requested
+`customer_email` | String! | Email of the person who created the return request
```

Comment:
After the recent changes `cusomter_name` and `customer_email` were replaced by the `customer` field that returns `ReturnCustomer` object.

Reply:
Also, it might make sense to add the `ReturnCustomer` documentation as well. The new type is declared here 

https://github.com/magento/partners-magento2ee/blob/6708b57704b9cd319774a0520ed6c8ceca796ca8/app/code/Magento/RmaGraphQl/etc/schema.graphqls#L110